### PR TITLE
Control: add localization_pose message into record for Control Channel

### PR DESCRIPTION
**Pull request by kevin-y-wang**

### CODE REVIEW

File: modules/control/controller/lat_controller.cc [hunk 1]
```diff
@@ -86,7 +86,9 @@ LatController::LatController() : name_("LQR-based Lateral Controller") {
   AINFO << "Using " << name_;
 }
 
-LatController::~LatController() { CloseLogFile(); }
+LatController::~LatController() {
+  CloseLogFile();
+}
```
Comment (xiaoxq) [inline modules/control/controller/lat_controller.cc]: These are auto-adjusted by clang-format tool. No need to change.

Comment (kevin-y-wang) [reply modules/control/controller/lat_controller.cc]: Fixed. Thanks.

**Comment by kevin-y-wang** [comment]
Move the localization pose into input debug per comment. Thanks

File: modules/control/control_component.cc [hunk 2]
```diff
@@ -219,6 +219,10 @@ Status ControlComponent::ProduceControlCommand(
     auto debug = control_command->mutable_debug()->mutable_input_debug();
     debug->mutable_localization_header()->CopyFrom(
         local_view_.localization.header());
+    debug->mutable_localization_pose()->mutable_position()->CopyFrom(
```
Comment (Capri2014) [inline modules/control/control_component.cc]: why copy them separately ?

**Pull request closed without merging**